### PR TITLE
fix sell coin issue

### PR DESCRIPTION
**Pull request by mubarak23**
PR base on issue: #47

### CODE REVIEW

**Comment by mubarak23** [comment]
@MSghais kindly review this,

llet me know what is missing

File: onchain/cairo/src/launchpad/launchpad.cairo [hunk 1]
```diff
@@ -1060,6 +1062,15 @@ mod LaunchpadMarketplace {
             let pool_coin = self.launched_coins.read(coin_address);
             let total_supply = pool_coin.total_supply.clone();
             let current_supply = pool_coin.token_holded.clone();
+            if (!is_decreased) {
```
Comment (MSghais) [inline onchain/cairo/src/launchpad/launchpad.cairo:1065]: I think it's not the correct condition, this is the formula for the is_decreased == true no? 

Also, can you delete the parenthesis? Not needed in Rust/Cairo I think

**Review by MSghais — CHANGES_REQUESTED** [review]
Added some comments to change

File: onchain/cairo/src/launchpad/launchpad.cairo (hunk 1 above)
Comment (mubarak23) [reply onchain/cairo/src/launchpad/launchpad.cairo:1065]: i see

**Pull request closed without merging**